__tests__/database/migration/file/index.test.ts

```diff
@@ -7,6 +7,6 @@ describe('TESTING all migration file', () => {
     // Execution
 
     // Assertion
-    expect(allMigrations.length).toEqual(8);
+    expect(allMigrations.length).toEqual(9);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test is updated to expect 9 migrations instead of 8. This change seems to be in line with the addition of a new migration for the `Chat` table. However, it's important to ensure that this number matches the actual number of migrations in your project.
```diff
-    expect(allMigrations.length).toEqual(8);
+    expect(allMigrations.length).toEqual(9);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->